projects/developer/src/app/common/components/recipe-graph/component.html

```diff
@@ -217,27 +217,27 @@
                                     <ul class="list-unstyled media-type-list" *ngIf="file.mediaTypes">
                                         <li *ngFor="let mediaType of file.mediaTypes">{{ mediaType }}</li>
                                     </ul>
-                                    <div class="connections"
-                                         *ngIf="selectedNodeConnections && selectedNodeConnections.length > 0">

```

Comment:
Is there a chance that selectedNodeConnections could be null?

Reply:
I'm not 100% sure on that; "selectedNodeConnections && selectedNodeConnections.length > 0" is Rachel's code; I'll ask her.